packages/zendesk-adapter/src/filters/field_references.ts

```diff
@@ -842,7 +842,15 @@ const commonFieldNameToTypeMappingDefs: ZendeskFieldReferenceDefinition[] = [
   // note: this overlaps with additional strategies, but because the first strategy
   // is chosen for serialization, it is safe
   {
-    src: { field: 'value', parentTypes: ['workspace__conditions__all'] },
+    src: {
+      field: 'value',
+      parentTypes: [
```

Comment:
this is not related to this PR right?

Reply:
No, talked with Neta about it (its a fix on the way)